docs/adr/0000-odh-release-policy.md

```diff
@@ -0,0 +1,39 @@
+# Open Data Hub Release Policy 
+
+* Status: proposed
+* Deciders: vpavlin 
+* Date: 2020-Nov-03 
+
+Technical Story: As an Open Source project, we want to document the policies and guideline on how we create a new
+release.
+
+## Context and Problem Statement
+
+The Open Data Hub itself consists of many components all having their own release cycles. ODH users might decide to
+updata individual components such as container images used by JupyterHub. Nevertheless it is required to create
```

Comment:
s/updata/update/

Reply:
@goern Can you fix this, so that I can merge?

Reply:
@goern One more nano typo